app/config/locale/templates/email-base.tpl

```diff
@@ -198,12 +169,11 @@
       }
 
       .btn-primary table td:hover {
-        background-color: {{bg-cta-hover}} !important;
+        opacity: 0.7 !important;
```

Comment:
https://www.campaignmonitor.com/css/positioning-display/opacity/

`opacity` has limited support for E-Mail clients.

Reply:
Thanks for the heads up, I think we can live with it.